scvi/hub/model_card_template.py

```diff
@@ -0,0 +1,45 @@
+template = """\
+---
+{card_data}
+---
+
+# Description
+
+{description}
+
+# Model properties
+
+Many model properties are in the model tags. Some more are listed below.
+
+**cell_count**: {cell_count}
+
+**gene_count**: {gene_count}
+
+**model_init_params**:
+```json
+{model_init_params}
+```
+
+**model_setup_anndata_args**:
+```json
+{model_setup_anndata_args}
+```
+
+**model_parent_module**: {model_parent_module}
+
+**data_is_latent**: {data_is_latent}
+
+# Large data
```

Comment:
```suggestion
# Full data
```

Reply:
I actually would rather not use full because it implies that there is a non-full data, which is not always the case (like in the scenario where you have a non latent model and just a data that is large). I also like large -- or something along those lines -- because it makes it more clear that that data is large. Full could mean anything. But I am open to other words than Large though.

Reply:
Training data?

Reply:
Added to https://github.com/scverse/scvi-tools/issues/1817
See https://github.com/scverse/scvi-tools/pull/1812#discussion_r1045147118